Pod/Classes/Forms/UIView+Lookup.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  UIView+Lookup.swift
+//  Swiftilities
+//
+//  Created by Brian King on 4/6/16.
+//  Copyright © 2016 Raizlabs. All rights reserved.
+//
+
+import Foundation
+
+extension UIView {
+
+    /// Lookup the subviews of a specific type, sorted by y position.
+    ///
+    /// - parameter includeHidden: Return hidden views in the response. The default is false.
+    /// - parameter includeNonInteractable: Return views that do not have user interaction enabled. The default is false.
+    ///
+    /// - returns: An array of views, of type T, sorted by y position.
+    public func lookupSortedViews<T: UIView>(includeHidden: Bool = false, includeNonInteractable: Bool = false) -> [T] {
```

Comment:
I think this may be helpful outside of the form library, and I never liked the name of this guy.

```
public func lookupSortedSubviews<T: UIView>(includeHidden includeHidden: Bool = false, includeNonInteractable: Bool = false) -> [T]
```
The 2 changes being `includeHidden` should be forced to specify a keyword, and second specifying `SortedSubviews` instead of `SortedViews`.  Any thoughts to make `includeNonInteractable` non cringe-inducing?

Changing the API to take a closure `filter: (view:T) -> Bool` may be a better choice too.

Also, we use this 'Type Injection' pattern a lot, where the return type is inferred by the call site. I almost feel like this method should be named `lookupSortedSubviewsOfType`. We should look at our other uses of this pattern and see if that suffix helps the clarity. Any thoughts here @ateliercw ?